source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/list/__tests__/pagination-acceptance-test.tsx

```diff
@@ -54,7 +54,7 @@ describe('Acceptance | Pagination', () => {
 
         navigateTo(screen.path);
 
-        it('Pagination is not displayed',() => {
+        xit('Pagination is not displayed',() => {
```

Comment:
why are these disabled?

Reply:
Because pagination is never hidden, you said some time ago that because of a bug in the JSON API we couldn't hide the pagination. So I don't how this test even worked.

Reply:
well, we can't just make changes without understanding why. :)

It's either was working due to a timing issue, or because the scenario / setup, is, indeed correct.

There are 0 projects -> Pagination is not displayed -- this makes sense.
There is less than one page worth of projects -> Pagination is not displayed -- also makes sense

Reply:
no, if it passes right now, just leave it enabled?

Reply:
yeah, but they are not passing, that is why I change them.

Reply:
Something is wrong with this test because as you can see on the screen it is passing when the pagination is been there

![bug](https://user-images.githubusercontent.com/501129/49331704-84069900-f56f-11e8-962e-9f8cb61126e4.png)

Reply:
Ok I found why is working in Master, that test is using isPresent to check if the pagination is there but what happen is that the table is not fully loaded when you enter to that test so is present return false correctly. If you use isVisible instead of isPresent the test breaks.

Reply:
So it sounds like the 'it' should be 'it.always'
And if the behavior isn't implemented, we can only allow it to be commented out with the proper documentation comments surrounding those tests. To protect future people from investigating again.